src/lang/lt.rs

```diff
@@ -373,17 +373,17 @@ pub static ref T: std::collections::HashMap<&'static str, &'static str> =
 		("Directory", "Katalogas"),
 		("Automatically record incoming sessions", "Automatiškai įrašyti įeinančius seansus"),
 		("Change", "Keisti"),
-		("Start session recording", "Pradėti seanso įrašymą"),
-		("Stop session recording", "Sustabdyti seanso įrašymą"),
-		("Enable Recording Session", "Įgalinti įrašymo seansą"),
-		("Allow recording session", "Leisti įrašymo sesiją"),
+		("Start session recording", "Pradėti seanso įrašinėjimą"),
+		("Stop session recording", "Sustabdyti seanso įrašinėjimą"),
+		("Enable Recording Session", "Įgalinti seanso įrašinėjimą"),
+		("Allow recording session", "Leisti seanso įrašinėjimą"),
 		("Enable LAN Discovery", "Įgalinti LAN aptikimą"),
 		("Deny LAN Discovery", "Neleisti LAN aptikimo"),
 		("Write a message", "Rašyti žinutę"),
 		("Prompt", "Užuomina"),
 		("Please wait for confirmation of UAC...", "Palaukite UAC patvirtinimo..."),
 		("elevated_foreground_window_tip", "Dabartinis nuotolinio darbalaukio langas reikalauja didesnių privilegijų, todėl laikinai neįmanoma naudoti pelės ir klaviatūros. Galite paprašyti nuotolinio vartotojo sumažinti dabartinį langą arba spustelėti aukščio mygtuką ryšio valdymo lange. Norint išvengti šios problemos ateityje, rekomenduojama programinę įrangą įdiegti nuotoliniame įrenginyje."),
-		("Disconnected", "Atjungtas"),
+		("Disconnected", "Atsijungęs, seanso laikas:"),

```

Comment:
`seanso laikas:`?

Reply:
This translates to "session time", but perhaps needs to be changed in the code, not in the translation file. I will fix it. I'm sorry.